packages/flutter_mobx_hooks/coverage/html/src/use_autorun.dart.func.html

```diff
@@ -0,0 +1,72 @@
+<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.01 Transitional//EN">
```

Comment:
Could you remove all folders besides `lib` and `test`, and add them to the gitignore?

Reply:
oops sure! I didn't notice they were committed sorry about that

Reply:
Should be all good now, don't know what append a lot of useless files where added in version source...